render/render.go

```diff
@@ -55,15 +56,18 @@ func getPolicyType(yaml string) string {
 		return mutate
 	} else if yamlContainsPolicy(newYAML, validate) {
 		return validate
-	} else {
+	} else if yamlContainsPolicy(newYAML, verifyImages) {
 		return verifyImages
+	} else {
+		return cleanUp
```

Comment:
I know this works, but honestly I think this is a bit lazy. Rather than assuming the type is always a cleanup policy by process of elimination, I would rather do this as a positive determination: If the apiVersion and kind say it's a cleanup policy then it's a cleanup policy. This is a different CRD than the `ClusterPolicy` and `Policy` which the render logic was set up to match. The reason this would be better is, if in the future, we want to start rendering Markdown from newer Kyverno CR types, we'll already have something to go on.

Reply:
Updated the code.